actions/user_actions.jsx

```diff
@@ -697,3 +697,17 @@ export async function sendPasswordResetEmail(email, success, error) {
         error({id: err.server_error_id, ...err});
     }
 }
+
+/**
+ * Send a getUserAnalytics request for a user (userId) and team id (teamId)
+ * Pass a string indicating which dataset to return (name)
+ *
+ * @param {string} userId - The target user for the requested analytics
+ * @param {string} teamId - The target team for the requested analytics
+ * @param {string} name - The name of the type of analytic data to return
```

Comment:
Are the valid inputs for `name` documented somewhere? Consider linking to that place, or consider documenting what inputs are valid here.